modules/programs/k9s.nix

```
@@ -8,9 +8,9 @@ let
   yamlFormat = pkgs.formats.yaml { };
 
 in {
-  meta.maintainers = with maintainers; [ katexochen liyangau ];
+  meta.maintainers = with maintainers; [ katexochen liyangau LucasWagler ];
```

To refer to the HM maintainers file you need to use `lib.hm.maintainers` attribute set:

```suggestion
  meta.maintainers = with maintainers; [ katexochen liyangau hm.maintainers.LucasWagler ];
```

The `lib.maintainers` attribute set refers to the maintainer list of Nixpkgs.